post-scan-actions/aws-python-promote-or-quarantine/template.yml

```diff
@@ -53,6 +53,23 @@ Parameters:
     Description: |
       The method by which files were quarantined.
       (Options: move, copy)
+  VpcSubnetId:
+    Type: String
+    Default: ''
+    Description: |
+      [Optional] The subnet ID of the VPC where the S3 bucket is located.
+      Leave it blank to disable using a VPC.
+  VpcSecurityGroupId:
+    Type: String
+    Default: ''
+    Description: |
+      [Optional] The security group ID of the VPC where the S3 bucket is located.
+      Leave it blank to disable using a VPC.
+  KMSKeyARN:
+    Type: String
```

Comment:
To support SSE-KMS enabled promote or quarantined buckets at the same time, suggest making it a [CommaDelimitedList](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) type parameter. So that the plugin can support when the buckets are encrypted with different keys.

Reply:
Done!